packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -319,7 +327,7 @@ const SelectableSearchInput = (props: TSelectableSearchInputProps) => {
     if (props.onSubmit) {
       props.onSubmit({
         text: searchValue,
-        option: selectedOption?.value,
+        option: selectedOption?.value ?? '',
```

Comment:
This needs to be double checked. The `option` field is apparently required but the `selectedOption?.value` can be `undefined`.
How should we handle this? Is it fine to return an empty string?

@commercetools/shield-team-fe